e2e-tests/cypress/support/appeal-header-footer/confirmGoogleAnalyticsLinkIsPresent.js

```diff
@@ -2,7 +2,7 @@ module.exports = () => {
   cy.get('head script[data-cy="Google Analytics"]').should(
     'have.attr',
     'src',
-    'https://www.googletagmanager.com/gtag/js?id=G-RTYZW789M0',
+    'https://www.googletagmanager.com/gtag/js?id=googleID',
```

Comment:
Does this need `$googleID` (I know nothing of Nunjucks)

Reply:
I read this as cypress looking for the fallback value as the env var isn't defined outside of prod.